### PR TITLE
Add Listenbrainz plugin

### PR DESCRIPTION
**Pull request by icyphox**
Implements `.np` and `.lbz`; where the latter is used for initial setup of Listenbrainz username.

### CODE REVIEW

File: db/db.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package db
+
+import (
+	"github.com/dgraph-io/badger/v3"
+)
+
+var DB *badger.DB
```
Comment (adedomin) [inline db/db.go:7]: if you feel paranoid about this, we could always do something like "dependency injection" from the registry point of view.

File: plugins/listenbrainz/nowplaying.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+func getListen(url string) (*ListenInfo, error) {
+	li := ListenInfo{}
+	r, err := http.Get(url)
+	if err != nil {
+		return nil, err
+	} else if r.StatusCode != 200 {
+		return nil, fmt.Errorf("non-200 response from Listenbrainz")
+	}
+
+	json.NewDecoder(r.Body).Decode(&li)
```
Comment (adedomin) [inline plugins/listenbrainz/nowplaying.go:37]: you need to close the body, no?

Comment (adedomin) [reply plugins/listenbrainz/nowplaying.go:37]: it is, io.ReadCloser

**Review by adedomin — COMMENTED** [review]
Look into the getListen function and the resp object closure.

File: db/db.go (hunk 1 above)
Comment (icyphox) [reply db/db.go:7]: I think this is _ok_ for now. We'll revisit it later.

**Pull request merged**